tools/java/org/unicode/cldr/tool/ChartDelta.java

```diff
@@ -1422,6 +1519,80 @@ private static boolean isHighLevelCurrencyName(String path, String locale) {
             return false;
         }
 
+        /**
```

Comment:
I think this will be more compact and easier to change if we use a regex lookup.

Reply:
OK, I'll use a regex.